docs/source/release/v6.6.0/diffraction.rst

```diff
@@ -10,60 +10,64 @@ Powder Diffraction
 
 New features
 ############
-- Introduce an input parameter in :ref:`WANDPowderReduction <algm-WANDPowderReduction>` to specify that the input workspaces are from event filtering. In such a situation, the overall reduction time could be reduced significantly.
+- Introduced an input parameter in :ref:`WANDPowderReduction <algm-WANDPowderReduction>` to specify that the input workspaces are from event filtering. In such a situation, the overall reduction time may be significantly reduced.
 - Added support for ILL D4C instrument, a liquid diffractometer. This adds a new algorithm responsible for the reduction stage: :ref:`D4ILLReduction <algm-D4ILLReduction>`, which handles calling the loader, correcting for dead time, bank position offsets, relative efficiency, and normalisation to monitor or time.
-- Increase the speed of the fourier filter in the POLARIS total scattering reduction by reducing the Rmax parameter used in the pair of (forward and backward) PDFFourierTransform calls and optimising the integration code inside :ref:`PDFFourierTransform v2 <algm-PDFFourierTransform-v2>`
+- Increased the speed of the fourier filter in the POLARIS total scattering reduction by reducing the ``Rmax`` parameter used in the pair of (forward and backward) ``PDFFourierTransform`` calls and optimising the integration code inside :ref:`PDFFourierTransform v2 <algm-PDFFourierTransform-v2>`.
```

Comment:
Would it be good to capitalise "Fourier" here?

Reply:
Huh, didn't know it was a name. Good catch